### DISCUSSION TITLE
Why do not support firefox?

### DISCUSSION THREAD

**Discussion by saintthor** [nexai-cloud/nexai.js#2] [General]
In firefox, I can't see the bot on my site and even can not open your website.

Don't ignore firefox. It is important.

**Comment by fijiwebdesign** [comment]
Hello @saintthor 
Thank you for notifying us about the issue. We will fix this immediately and get back to you.

**Comment by fijiwebdesign** [comment]
@saintthor 

**I've added fixes to support Firefox in nexai.js version 0.0.3-2** 

Apologies for the oversight - the Speech API does not work with FF and support for this will be added later. This prevented the page from loading due to the uncaught error within the speech api. 

Please update your nexai.js version in `package.json`: 

```
"nexai.js": "0.0.3-2",
```

Then reinstall nexai.js 

```
npm install 
// yarn install
// pnpm install
// etc.
```

You can also do a one liner:

```
yarn remove nexai.js && yarn install nexai.js 
```

The website and chat panel should load properly in Firefox now.

**Reply by saintthor** [reply]
I can open your website in firefox now. But in my page it still not work.

Have you updated [https://nexai.site/ai/embed.umd.js](view-source:https://nexai.site/ai/embed.umd.js) ? I added it in my <html><script></script></html>.

BTW, it is difficault to fine the code in you website.
